### ISSUE TITLE
Issue when importing MAGIK

### ISSUE THREAD

**Issue by BptGrm** [softmatterlab/DeepTrack-2.0#103]
When trying the MAGIK example [notebook](https://colab.research.google.com/github/softmatterlab/DeepTrack-2.0/blob/develop/examples/MAGIK/cell_migration_analysis.ipynb), it fails when importing the modules : 
```
ModuleNotFoundError                       Traceback (most recent call last)
[<ipython-input-8-bcd0be12bc00>](https://localhost:8080/#) in <module>()
      1 import deeptrack
      2 print(dir(deeptrack.models))
----> 3 from deeptrack.models.gnns.generators import GraphGenerator
      4 
      5 

ModuleNotFoundError: No module named 'deeptrack.models.gnns.generators'
```
When printing `dir(deeptrack.models)`, it does not list "GNN".
I also noticed that [deeptrack/models/__init__.py](https://github.com/softmatterlab/DeepTrack-2.0/blob/develop/deeptrack/models/__init__.py) does not list `from .gnns import *`, could that be the issue ?

**Comment by JesusPinedaC** [comment]
Hello @BptGrm!

Thanks for letting us know about the issue with the tutorial!

The error comes from the fact that MAGIK was not yet available via pip install. However, we´ve released deeptrack 1.1.0, which contains all the features of MAGIK. 

The problem should be solved by updating your version of deeptrack!

**Comment by BptGrm** [comment]
Works on Colab, thanks !
